scripts/kind-e2e/lib_armada_deploy_kind.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# This should only be sourced
+if [ "${0##*/}" = "lib_helm_deploy_subm.sh" ]; then
+    echo "Don't run me, source me" >&2
+    exit 1
+fi
+
+### Variables ###
+
+# FIXME: Armada support for setting pod/service CIRDs via flags?
```

Comment:
I would keep the CIDRs may be as part of the top level e2e.sh or any other script where we set variables with the deployment details.

Reply:
I'm not sure I understand this one. The CIDRs vars are in different libs because they have different values for the two deployment methods. They can't be extracted to a common place like e2e.sh without naming them per-deployment-method, and breaking the model of using libs with parallel vars/fns names and no conditionals in common code.